utils.go

```diff
@@ -15,23 +15,16 @@ func isGenericKey(key string) bool {
 
 func isIgnoredKey(key string) bool {
 	ignoredKeys := []string{"BITRISEIO_PULL_REQUEST_REPOSITORY_URL"}
-
-	for _, ignoredKey := range ignoredKeys {
-		if key == ignoredKey {
-			return true
-		}
-	}
-	return false
+	return sliceutil.IsStringInSlice(key, ignoredKeys)
 }
 
 func splitEnv(env string) (string, string) {
 	e := strings.Split(env, "=")
-	return e[0], strings.Join(e[1:], "=")
-}
 
-func failf(f string, args ...interface{}) {
-	log.Errorf(f, args...)
-	os.Exit(1)
+	if len(e) == 0 {
+		return e[0], "="
```

Comment:
`if len(e) == 0` this would fail, you try to refer the first index in an empty slice

Reply:
other than that the original solution was fine, the 2. return value may look a bit strange  `strings.Join(e[1:], "=")`, but it is correct as:
- len(e) >= 1 always
- slice indexing: `0 <= low <= high <= len(e)`
so `e[1:]` works for a slice which has 1 element

Reply:
@BirmacherAkos yeh, @godrei is right. I was not aware of this `[1:]` correctly as fas as I see. Even `fmt.Println("."[1:])` works fine.

Reply:
Fixed in:
https://github.com/bitrise-steplib/steps-generic-file-storage/pull/4/commits/b334dbea37fc4b3a0725791eff196741b623065d